src/Synthesiser.cpp

```diff
@@ -365,8 +365,6 @@ void Synthesiser::emitCode(std::ostream& out, const RamStatement& stmt) {
         void visitClear(const RamClear& clear, std::ostream& out) override {
             PRINT_BEGIN_COMMENT(out);
 
-            out << "if (!isHintsProfilingEnabled()"
-                << (clear.getRelation().isTemp() ? ") " : "&& performIO) ");
```

Comment:
`performIO` has been removed, which means that the relation will always be purged here which then breaks embedded souffle